content/applications/services/helpdesk/advanced/close_tickets.rst

```diff
@@ -15,46 +15,46 @@ As work on a ticket progresses, it is moved along to the next stage in the pipel
    :align: center
    :alt: View of stage on Helpdesk pipeline with emphasis on gear icon and edit stage option.
 
-.. warning::
-   Clicking the gear icon also displays the option to :guilabel:`Fold` the stage. This setting folds
-   the stage *temporarily* to simplify the kanban view. This does *not* close the tickets in this
-   stage. It also does not permanently fold the stage. If a stage needs to be folded so the tickets
-   can be marked as closed, continue following the steps below.
-
-From the menu that appears, select :guilabel:`Edit Stage`. This will open the stage's settings.
-Check the box labeled :guilabel:`Folded in Kanban` towards the top of the window, and then
-:guilabel:`Save & Close` to confirm the changes. Now, tickets that reach this stage will be
-considered as *closed*.
+From the menu that appears, select :guilabel:`Edit`. This opens the stage's settings. Check the box
+labeled :guilabel:`Folded in Kanban` towards the top of the window, and then :guilabel:`Save &
+Close` to confirm the changes. Tickets that reach this stage are now considered as *closed*.
 
    .. image:: close_tickets/closing-folded-setting.png
       :align: center
       :alt: Stage settings page.
 
+.. warning::
+   Clicking the :guilabel:`⚙️ (gear)` icon also displays the option to :guilabel:`Fold` the stage.
+   This setting folds the stage *temporarily* to simplify the Kanban view. This does **not** close
+   the tickets in this stage. It also does **not** permanently fold the stage. If a stage needs to
+   be folded so the tickets can be marked as closed, the :guilabel:`Folded in Kanban` checkbox
+   **must** be checked on the stage's settings.
+
 Automatically close inactive tickets
 ====================================
 
 Tickets that are inactive for a set period of time can be automatically closed. At that point, they
-will be moved to a folded stage.
+are moved to a folded stage.
 
-Go to the team's settings page by going to :menuselection:`Helpdesk --> Configuration --> Teams`.
-Under the :guilabel:`Self-Service` section, enable :guilabel:`Automatic Closing`.
+Go to the team's settings page by going to :menuselection:`Helpdesk app --> Configuration -->
+Helpdesk Teams`. Under the :guilabel:`Self-Service` section, enable :guilabel:`Automatic Closing`.
 
-If one of the team's stages is set to be folded in the kanban view, it will be the default selection
-in the :guilabel:`Move to Stage` field. If the team has more than one folded stage, the stage that
-occurs first in the pipeline will be the default. If no stage is folded, the default selection will
-be the last stage in the pipeline.
+If one of the team's stages is set to be folded in the Kanban view, the folded stage is the default
+selection in the :guilabel:`Move to Stage` field. If the team has more than one folded stage, the
+folded stage that occurs first in the pipeline is the default. If no stage is folded, the default
+selection is the last stage in the pipeline.
```

Comment:
```suggestion
After ticking the box for :guilabel:`Automatic Closing`, three new fields appear beneath:

- :guilabel:`Move to Stage`
- :guilabel:`After (#) days of inactivity`
- :guilabel:`In Stages`

If one of the team's stages is set to be folded in the Kanban view, the folded stage is the default
selection in the :guilabel:`Move to Stage` field. If the team has more than one folded stage, the
folded stage that occurs first in the pipeline is the default. If no stage is folded, the default
selection is the last stage in the pipeline.
```
just for some more detail re: the fields that appear after ticking the box for that feature